recipe/meta.yaml

```diff
@@ -15,6 +15,9 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
```

Comment:
those requirements shouldn't be needed for libgdal

Reply:
It is a bug in the migrator. Also, aren't we already building osx-arm64? I believe this PR may not be needed.

Reply:
It just showed up because of the creation of the v3.6.x branch yesterday.